pkg/network/event_common.go

```diff
@@ -166,7 +166,7 @@ func (c ConnectionStats) String() string {
 //     4B      2B      2B     .5B     .5B      4/16B        4/16B   = 17/41B
 //    32b     16b     16b      4b      4b     32/128b      32/128b
 // |  PID  | SPORT | DPORT | Family | Type |  SrcAddr  |  DestAddr
-func (c ConnectionStats) ByteKey(buf [ConnectionByteKeyMaxLen]byte) ([]byte, error) {
+func (c ConnectionStats) ByteKey(buf []byte) ([]byte, error) {
```

Comment:
The reason this was an array was to prevent the caller from using this function incorrectly. `copy` relies on the slice already being the correct length. If you want to keep this a slice, maybe we should do a `len` check and return an error if it is too small?